apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/ExperimentsResource.java

```diff
@@ -132,6 +133,20 @@ public Response create(
         return Response.created(uri).build();
     }
 
+    @DELETE
+    @Path("/{id}")
+    @Operation(operationId = "deleteExperimentById", summary = "Delete experiment", description = "Delete experiment", responses = {
+            @ApiResponse(responseCode = "204", description = "No content")})
+    public Response deleteExperimentById(@PathParam("id") UUID id) {
```

Comment:
Let's better just have a single batch endpoint that takes a set of ids to be deleted. If people wants to delete a single experiment, they just need to pass an ID.